fiftyone/utils/ultralytics_internal.py

```
@@ -0,0 +1,126 @@
+        self.eval()
+
+
+class UltralyticsMobileCLIPTS(BaseMobileCLIP):
+    """Adaptor for ultralytics.nn.text_model.MobileCLIPTS.
+
+    Ensures that CLIP model is saved to FiftyOne model zoo directory.
+    """
+
+    def __init__(self, device):
+        text_model.TextModel.__init__(self)
+        from ultralytics.utils.downloads import attempt_download_asset
```

Ruff (0.11.9)

`ultralytics.utils.downloads.attempt_download_asset` imported but unused

Remove unused import: `ultralytics.utils.downloads.attempt_download_asset`

(F401)